mindsdb/api/mongo/classes/query_sql.py

```diff
@@ -1,27 +1,40 @@
-from mindsdb.api.mysql.mysql_proxy.controllers.session_controller import SessionController
-from mindsdb.api.mysql.mysql_proxy.executor.executor_commands import ExecuteCommands
+from mindsdb.api.mysql.mysql_proxy.controllers import SessionController
+from mindsdb.api.mysql.mysql_proxy.executor import Executor
+
+
+class SqlServerStub:
+    """This class is just an emulation of Server object,
+    used by Executor.
+    In 'monolithic' mode of MindsDB work the Executor takes
+    some information from the sql server which. Here we emulate
+    this object."""
+
+    def __init__(self, **kwargs):
+        for arg in kwargs:
+            setattr(self, arg, kwargs[arg])
 
 
 def run_sql_command(mindsdb_env, ast_query):
     sql_session = SessionController()
     sql_session.database = 'mindsdb'
+    sqlserver = SqlServerStub(connection_id=-1)
 
-    command_executor = ExecuteCommands(sql_session, executor=None)
-    ret = command_executor.execute_command(ast_query)
-    if ret.error_code is not None:
-        raise Exception(ret.error_message)
+    executor = Executor(sql_session, sqlserver)
+    executor.query_execute(ast_query.to_string())
```

Comment:
it doesn't keep datatypes, for example datetime becomes string.
need to implement ast serialize/deserialize

Reply:
Fixed